packages/handbrake.rb

```diff
@@ -25,7 +34,8 @@ def self.patch
 
   def self.build
     ENV['TMPDIR'] = "#{CREW_PREFIX}/tmp"
-    system "./configure --prefix=#{CREW_PREFIX} --enable-x265 --enable-numa --enable-fdk-aac --harden"
+    system "env CFLAGS='-pipe -flto=auto' CXXFLAGS='-pipe -flto=auto' \
+      ./configure --prefix=#{CREW_PREFIX} --enable-x265 --enable-numa --enable-fdk-aac --harden"
```

Comment:
I assume `#{CREW_OPTIONS}` fails?

Reply:
I didn't try. This was a quick edit just in case it was your build environment which was broken...